tests/foreman/cli/test_registration.py

```diff
@@ -190,6 +190,7 @@ def test_negative_register_twice(module_ak_with_cv, module_org, rhel_contenthost
     assert 'This system is already registered' in str(result.stderr)
 
 
+@pytest.mark.pit_client
```

Comment:
we are testing force registration in "test_global_registration_form_populate" so we can skip this one and we can test "test_positive_allow_reregistration_when_dmi_uuid_changed" instead for pit.
Wdyt @ogajduse ?

Reply:
Agreed, marked `test_positive_allow_reregistration_when_dmi_uuid_changed` for pit